src/linear_algebra.jl

```diff
@@ -75,20 +75,29 @@ function A_b(eq, var, check)
 end
 
 """
-    solve_for(eqs::Vector, vars::Vector; simplify=true, check=true)
+    solve_for(eqs, vars; simplify=true, check=true)
 
-Solve the vector of equations `eqs` for a set of variables `vars`.
+Solve equation(s) `eqs` for a set of variables `vars`.
 
 Assumes `length(eqs) == length(vars)`
 
 Currently only works if all equations are linear. `check` if the expr is linear
 w.r.t `vars`.
 """
-function solve_for(eqs, vars; simplify=true, check=true)
+function solve_for(eq, var; simplify=true, check=true) # scalar case
```

Comment:
Can we use `simplify=false` by default? Seems like the canonicalization should get to most of the required simplification.

Reply:
Yes.